lib/aca_entities/families/family_member_reference.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module AcaEntities
+  module Families
+    class FamilyMemberReference < Dry::Struct
+      # TODO: Person hbx_id until we add hbx_id on family member
+      attribute :hbx_id,          Types::String.optional.meta(omittable: false)
+    end
```

Comment:
Add attributes: 

1. person_hbx_id
2. person_first_name
3. person_last_name

Reply:
Added additional attributes to family member reference.